strapi/strapi
# Track settings pages usage

### What does it do?

Adds a `willNavigate` telemetry event when a user clicks on a link in the settings menu

### Why is it needed?

To track the usage of the audit logs page for users who did not disable telemetry, and offer that possibility for other settings pages as well

### How to test it?

- Open the settings page
- Make sure your browser's devtools are open and on the network page
- Click on the audit logs link
- You should see a "track" request to `https://analytics.strapi.io/api/v2/track`, its payload should be an object with an event property of `willNavigate`, and an eventProperties object with `/settings/audit-logs?pageSize=50&page=1&sort=date:DESC` as the destination
